support-frontend/assets/components/checkmarkList/checkmarkList.tsx

```diff
@@ -101,7 +113,24 @@ export function CheckmarkList({
 							<ChecklistItemIcon checked={item.isChecked} style={style} />
 						</div>
 					</td>
-					<td css={[checkListTextCss, item.maybeGreyedOut]}>{item.text}</td>
+					<td css={[checkListTextCss, item.maybeGreyedOut]}>
+						{typeof item.text === 'string' ? (
+							<span>
+								{item.text}
+								{item.toolTip && (
+									<Tooltip
+										cssOverrides={toolTipCss}
+										children={<p>{item.toolTip}</p>}
+										xAxisOffset={108}
+										yAxisOffset={12}
```

Comment:
I can't fully see why these aren't calculated by the tooltip, any why they are these specific values?

Looking at this a little closer - it looks like this offset if broken anyway:
![Screenshot 2024-01-25 at 09 33 58](https://github.com/guardian/support-frontend/assets/31692/f1bce3f3-d4b7-4059-a7e4-e3ced30dfd22)

It also looks like the only place we use it is [on the "Cancel anytime" prompt](https://github.com/guardian/support-frontend/blob/b115bdc88c53c40829a6cf9bac1e144f0b217377/support-frontend/assets/pages/supporter-plus-landing/formSections/amountAndBenefits.tsx#L130) - and [we never actually turn it on](https://github.com/guardian/support-frontend/blob/b115bdc88c53c40829a6cf9bac1e144f0b217377/support-frontend/assets/pages/supporter-plus-landing/twoStepPages/firstStepLanding.tsx#L162-L169).

As these values work, this is good.

Outside of this PR we might want to look at how we can always have the tooltip be next to the ℹ️ icon.

Reply:
Yep, agree its not perfect, here it works with the `xAxisOffset` but would be good for the  the floating container, rather than centering horizontally, was more intelligent to move over the ℹ️ icon. Vertically (`yAxisOffset`) we could default allowing no adjustment. Created PR here -> https://trello.com/c/OwGnNCjF/553-tooltip-vertical-and-horizontal-alignment-corrections